### ISSUE TITLE
VAM controller issue

### ISSUE THREAD

**Issue by diuxi2563** [korejan/ALVR#52]
## Description
When my headset torwards to the model, the controllers will drop off to the bottom of the screen. i can press the button but cannot move it. Then i need to click the menu button to reactive the controller. but around 3-5 sec it will drop to the bottom again.
I Tried to remove the model or back-facing to the model , controller will not drop to the bottom any more.
This situation only happens in alxr. I try it in ALVR.APK the controller is working normal.

## Environement
WIN10 

**CPU**:
9400F

**GPU**:
RTX2060

### Installation
ALXR v0.10.0
ALVR v18.2.3

**SteamVR Version**:
1.24.6

**Install Type**:
EXE

**OS Name and Version** (`winver` on Windows or `grep PRETTY_NAME /etc/os-release` on most Linux distributions):
Windows

**Comment by korejan** [comment]
hi, does this happen the with latest alxr [nightly builds](https://github.com/korejan/ALXR-nightly)?

**Comment by diuxi2563** [comment]
Yes, it still happens with ALXR  v0.10.0+nightly.2022.10.29 but much better than before. Now the controllers will reactive automatically. 
Upload the video for your easy reference.   In the video, i just move the controller randomly and you could see the line came from the controller will drop to the bottom as i said.
https://user-images.githubusercontent.com/117051161/199002197-a155e501-2774-4748-bed6-89f30a7bfab2.mp4
